src/audio-view.js

```diff
@@ -0,0 +1,193 @@
+/*
+  Game event               Sound                      Logic
+
+  Enter alone mode         Alone_Mode/Pulse_amb_loop  FIXME
+  Handshake                Alone_Mode/Hand_Shake_01   changes.handshakes (Set of bool)
+
+Mole:
+  Panel activated          Game_01/G01_LED_XX
+  Active panel touched     Game_01/G01_Success_01
+  Non-active panel touched Game_01/G01_Negative_01
+
+Disk:
+
+Simon:
+  Panel activated during pattern animation
+                           Game_03/G03_LED_XX
+  Correct panel touched
+                           Game_03/G03_LED_XX
+  Wrong panel touched
+                           Game_03/G03_Negative_01
+  Won level (after short delay)
+                           Game_03/G03_Success_01
+  Won all levels (after short delay)
+                           Game_03/G03_Light_Show_01
+
+*/
+
+const _ = require('lodash');
```

Comment:
I expected lodash entry points to be picked out by browserify, including only the actually used code paths in to our build code. Is that not the case?